dronegen/push.go

```diff
@@ -164,16 +164,13 @@ func pushPipeline(b buildType) pipeline {
 func sendErrorToSlackStep() step {
 	return step{
 		Name:  "Send Slack notification",
-		Image: "plugins/slack",
+		Image: "plugins/slack:1.4.1",
 		Settings: map[string]value{
 			"webhook": {fromSecret: "SLACK_WEBHOOK_DEV_TELEPORT"},
 			"template": {raw: `*{{#success build.status}}✔{{ else }}✘{{/success}} {{ uppercasefirst build.status }}: Build #{{ build.number }}* (type: ` + "`{{ build.event }}`" + `)
```

Comment:
consider moving the template to its own file and using go embedding to read it into a string. Then you don't need to worry about all the different quoting and concatenating strings.